code/__HELPERS/mobs.dm

```diff
@@ -298,6 +298,8 @@ Proc for attack log creation, because really why not
 	var/endtime = world.time + delay
 	var/starttime = world.time
 	. = 1
+	var/mob/living/L = user
+	var/is_living = istype(L)

```

Comment:
It's not about pretending non-living can have stuns, it's avoiding a typecheck and typecasting that can actually be avoided without any issue.